wiki/osu!_Program_Files/en.md

```diff
@@ -1,73 +1,66 @@
+osu! Program Files
+===================
+
 ![osu! Program Files in nutshell](Osu!sys.jpg "osu! Program Files in nutshell")
 
 Location
-========
+--------
 
-Windows
--------
+### Windows
 
 | Current default filepath | Old installer filepath |
 | ------------------------ | ---------------------- |
 | C:\Users\<Username>\AppData\Local\osu! | 	C:\Program Files\osu! / C:\Program Files(x86)\osu! (for 7 64-bit or 8) |
 > "C:" is your harddisk with osu! installed.
 
-![Folder](Folder.png "Folder") Folders
-===============================================================================================
+Folders
+------------------------------------------
 
-_MACOSX
---------
+### \_MACOSX
 
 **osu! for Macintosh OS.** For Windows OS users, just ignore it or delete it if you wish.
 
-Chat
-----
+### Chat
 
 Only appears when you used "/savelog" at Chat Console once or if you enabled "Automatically log private messages" in Options. The folder shows your **Chat Console tab's log of comments.**
 
 The styling is `{Tab_name}-{YYYYMMDD}-{HHMMSS}` in .txt format, which basically can be opened in any word application of choice(Notepad).
 
 **Example:** #multiplayer-20121115-040845 (/savelog at #multiplayer tab in 15th November 2012 at 0408hrs and 45secs).
 
-Downloads
----------
+### Downloads
 
 Appears when you had used the "osu!direct" download service. **Holds the .osz files which you are currently downloading from osu!direct**, later to be transferred into osu!'s Songs folder as legitimate beatmaps upon completion.
 
-Exports
--------
+### Exports
 
-Appears if you used the [Skin Selector's "Export as .osk"](Options#Skin_Selector "wikilink") or [Beatmap Editor's "Export Package"](Beatmap_Editor/Menu#File "wikilink"). **Holds .osz (beatmap)/.osk (skin) file you had exported from osu!.** [For how-to, click me](Osu!_File_Formats#From_the_game "wikilink").
+Appears if you used the [Skin Selector's "Export as .osk"](/wiki/Options) or [Beatmap Editor's "Export Package"](/wiki/Beatmap_Editor/Menu). **Holds .osz (beatmap)/.osk (skin) file you had exported from osu!.** [For how-to, click me](/wiki/osu!_File_Formats).
 
-Localisation
-------------
+### Localisation
 
 Appears when you had switched your Language under Graphics at Options page. **Holds translated text files which can be used each to replace the usual English text based on selected localisation.**
```

Comment:
change to:
```
This folder appears when you had switched your Language under Graphics in the Options. **It will hold the translated text files which can be used each to replace the usual English text based on the user's selected localisation.**
```